ocs_ci/ocs/pgsql.py

```diff
@@ -338,6 +342,55 @@ def respin_pgsql_app_pod(self):
             resource=app_pod, state=constants.STATUS_RUNNING, timeout=300
         )
 
+    def get_node_utilization(self, node_type=('worker', 'master')):
```

Comment:
@PrasadDesala do we want to capture this in the google sheet?...

Reply:
@PrasadDesala @hnallurv 
I think that we can print a table in the log file and export the data to google sheet.
What do you think?

Reply:
Yes, will make the changes once this PR is merged.